### DISCUSSION TITLE
Lower JsonSerializableNormalizer priority

### DISCUSSION THREAD

**Discussion by aprat84** [symfony/symfony#46661] [Q&A]
I use `Carbon` in my project to replace php native `DateTime`. `Carbon` implements `JsonSerializable` interface.

Since `JsonSerializableNormalizer` has higher priority than `DateTimeNormalizer`, I can't control the output format.

Shouldn't more generaliztic normalizers have less priority than more specific ones? In practice, `JsonSerializableNormalizer`'s priority be between -920 and -990?

<img width="1272" alt="image" src="https://user-images.githubusercontent.com/39481795/173331774-b9332c23-f9cb-4df7-a689-9e1d43c4fd1f.png">

**Comment by lyrixx** [comment]
That's a valid point IMHO. Can you open a PR? Thanks
